rasa/data.py

```diff
@@ -146,13 +146,25 @@ def is_story_file(file_path: Text) -> bool:
     Returns:
         `True` if it's a story file, otherwise `False`.
     """
-    _is_story_file = False
 
-    if file_path.endswith(".md"):
-        with open(file_path, encoding=DEFAULT_ENCODING) as f:
-            _is_story_file = any(_contains_story_pattern(l) for l in f)
-
-    return _is_story_file
+    if not file_path.endswith(".md"):
```

Comment:
This (module) function should be tested (said the one who wrote most of it 🙄 )

Reply:
added some tests :+1: